Automattic/sensei
# Update slugs for messages, lessons and questions

Fixes #3218.

### Changes proposed in this Pull Request
* Updates the archive slugs for lessons and questions to their plural form.
* Updates the single post type slug for a message to its singular form.

Note that these changes won't be applied until the user visits their _Permalinks_ settings page.

Although technically correct, these changes are also not without risk, as it means that any existing links to the old pages will return a 404.

### Testing instructions

* In WP admin, go to _Settings_ > _Permalinks_.
* On the front-end, browse to `yourdomain/lessons` and `yourdomain/questions`.
* Ensure the lesson and question archives are displayed, respectively.
* Browse to `yourdomain/messages` and ensure you see a list of your messages.
* Click on a single message and ensure the slug is singular (i.e. `message`).
